Mobile/starter_project_dartsmiths/lib/features/example/presentation/widgets/article_reading.dart

```diff
@@ -0,0 +1,127 @@
+                        ],
+                      ),
+                      const SizedBox(
+                        height: 20,
+                      ),
+                      const Text(
+                        'Four Things Everyone Needs To Know',
+                        style: TextStyle(
+                          fontWeight: FontWeight.bold,
+                          fontSize: 25,
+                        ),
+                      ),
+                      const SizedBox(
+                        height: 20,
+                      ),
+                      Row(
+                        mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                        children: [
+                          Row(
+                            children: [
+                              ClipRRect(
+                                borderRadius: BorderRadius.circular(18),
+                                child: Image.asset(
+                                  'images/fashion1.jpg',
+                                  width: 40,
+                                  height: 40,
+                                ),
+                              ),
+                              const SizedBox(
+                                width: 15,
+                              ),
+                              Column(
+                                crossAxisAlignment: CrossAxisAlignment.start,
+                                children: const [
+                                  Text(
+                                    'Richard Gervan',
+                                    style: TextStyle(color: Colors.blue),
+                                  ),
+                                  Text(
+                                    '2m ago',
+                                    style: TextStyle(color: Colors.blue),
+                                  )
+                                ],
+                              ),
+                            ],
+                          ),
+                          const Icon(Icons.bookmark_add_outlined)
+                        ],
+                      ),
+                    ],
+                  ),
+                ),
+                Container(
+                  decoration: BoxDecoration(
+                    borderRadius: BorderRadius.circular(20),
+                  ),
+                  height: 190,
+                  width: double.infinity,
+                  child: ClipRRect(
+                    borderRadius: const BorderRadius.only(
+                        topLeft: Radius.circular(20),
+                        topRight: Radius.circular(20)),
+                    child: Image.asset(
+                      'images/fashin3.jpg',
+                      width: double.infinity,
+                      fit: BoxFit.cover,
+                    ),
+                  ),
+                ),
+                const Padding(
+                    padding: EdgeInsets.fromLTRB(30, 25, 30, 0),
+                    child: Text(

```

Comment:
Adding text like this can make it harder while integrating. Declare a variable with the text content and assign the variable here

Reply:
Done

Reply:
Done